python/dgl/heterograph.py

```diff
@@ -2576,6 +2576,55 @@ def incidence_matrix(self, typestr, ctx=F.cpu(), etype=None):
     # Alias of ``incidence_matrix``
     inc = incidence_matrix
 
+    def line_graph(self, backtracking=True):
```

Comment:
same thing here. I would prefer to write a global function.
```
lg = dgl.line_graph(g)
```
And if we want to also have a syntax as member function. We could have something like this:
```python
def line_graph(g, ...):
    # implementation

DGLGraph.line_graph = line_graph
```

Reply:
Move to transform.py